### PR TITLE
500 error

### PR DESCRIPTION
**Pull request by alvinsiu**
#### What are the relevant tickets?
#43 

#### What's this PR do?
Fixes 500 error

### CODE REVIEW

**Comment by pdpinch** [comment]
Can I get some explanation about how this addresses the error I was seeing?

It looks like maybe I hadn't configured the LTI properly in my course?

**Comment by alvinsiu** [comment]
The fix is in commit https://github.com/mitodl/sga-lti/commit/2d7aec7e53bb86d13b25299a96ba7862178dbe30.  The issue was a backend issue, not an LTI configuration issue.  In the view_assignment view function, grader.students.filter(...) returns Student objects, but the rest of the function expected the User objects attache to those students to be returned.

**Comment by pdpinch** [comment]
@giocalitri can you take a quick look at this? I'd like to get it merged so I can continue with my QA of this application.

File: sga/middleware.py [hunk 1]
```diff
@@ -17,30 +17,38 @@ class SGAMiddleware(object):
     Middleware for processing incoming LTI requests
     """
     ADMIN_ROLES = ["Administrator", "Instructor"]
+    LTI_MIDDLEWARE_NOT_INSTALLED_MESSAGE = "LTI middleware not installed"
+    UNSUCCESSFUL_LTI_AUTHENTICATION_MESSAGE = "Bad or missing LTI credentials"
+    NO_CONTEXT_ID_MESSAGE = "No context_id in LTI parameters"
+    NO_RESOURCE_LINK_ID_MESSAGE = "No resource_link_id in LTI parameters"
+    REQUEST_USERNAME_FALSE_MESSAGE = "\"Request user's username\" must be set to True on this assignment."
```
Comment (giocalitri) [inline sga/middleware.py]: you can use the single quote `'` here and avoid the backslash

File: sga/middleware.py [hunk 2]
```diff
@@ -17,30 +17,38 @@ class SGAMiddleware(object):
...
         """
         Processes incoming LTI requests
         """
         if not hasattr(request, "LTI"):
-            raise ImproperlyConfigured("LTI middleware not installed")
+            raise ImproperlyConfigured(self.LTI_MIDDLEWARE_NOT_INSTALLED_MESSAGE)
         if "course_roles" not in request.session:
             request.session["course_roles"] = {}
 
         if request.lti_initial_request:
```
Comment (giocalitri) [inline sga/middleware.py]: it looks like the rest of the function is under this `if` block: maybe you should `return` if the `request.lti_initial_request` is `False` and move the rest of the code on the left-er block.

**Comment by giocalitri** [comment]
a couple of minor comments, but it seems fine overall (but I did not check functionality)

**Pull request merged**